ocs_ci/ocs/utils.py

```diff
@@ -916,6 +918,32 @@ def run_must_gather(log_dir_path, image, command=None):
     return mg_output
 
 
+def get_helper_pods_output():
+    """
+    Get the output of "oc describe mg-helper pods"
+
+    Returns:
+        output_describe_mg_helper (str): the output of "oc describe mg-helper pods"
+
+    """
+    from ocs_ci.ocs.resources.pod import get_pod_obj, get_pod_logs
```

Comment:
is this imported here for preventing a circular import?

Reply:
yes